NETCORE/.vsts/linux-build.yml

```diff
@@ -21,6 +23,12 @@ steps:
   inputs:
     version: 3.1.301
     
+- task: UseDotNet@2
+  displayName: install dotnet core 5
```

Comment:
@eddynaka Its donet 5, not dotnet core 5 :D

Reply:
updated!